src/System.IO.Pipelines/src/System/IO/Pipelines/PipeScheduler.cs

```diff
@@ -26,5 +28,27 @@ public abstract class PipeScheduler
         /// Requests <paramref name="action"/> to be run on scheduler with <paramref name="state"/> being passed in
         /// </summary>
         public abstract void Schedule(Action<object> action, object state);
+
+        internal virtual void UnsafeSchedule(Action<object> action, object state)
```

Comment:
This will make performance worse by default until all schedulers override.  We should just call Schedule

Reply:
Done